pynest/nest/tests/test_sp/test_disconnect.py

```diff
@@ -52,12 +54,6 @@ def setUp(self):
             'stdp_dopamine_synapse_hpc_lbl',
             'gap_junction',
             'gap_junction_lbl',
-            'diffusion_connection',
-            'diffusion_connection_lbl',
-            'rate_connection_instantaneous',
-            'rate_connection_instantaneous_lbl',
-            'rate_connection_delayed',
-            'rate_connection_delayed_lbl'
```

Comment:
Removing these makes the test fail because these synapse-models aren't supported by `iaf_neuron`/`iaf_psc_alpha`.

Reply:
Confirming - these need to be checked and added back to the exclude list